### PR TITLE
Added Pixabay image service. Fixes #1114

### LINKED ISSUE #1114: Adding image search service (with test and example)

### PR DESCRIPTION
**Pull request by brollb**

### CODE REVIEW

**Comment by brollb** [comment]
We should probably add a generic `getImage` with the argument `url` to this service @hamidzr

**Comment by brollb** [comment]
We also need to make sure we are not violating any terms of service with pixabay

**Comment by hamidzr** [comment]
pixabay requires us to show that we are using pixabay. should we add a credits method wich returns pixabay ? :)

**Comment by brollb** [comment]
what if we just change the service name to Pixabay? That should show that we are getting the images from them...

**Comment by brollb** [comment]
Can you change the name of the example and service to Pixabay?

File: src/server/rpc/procedures/pixabay/pixabay.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        return this._sendStruct(encodeQueryOptions(keywords, null, minHeight, item), parserFnGen(maxHeight));
+    };
+});
+
+pixabay.getImage = function(url){
+    return this._sendImage({queryString: '', baseUrl:url});
+};
+
+if(!process.env.PIXABAY) {
+    console.log('Warning: environment variable PIXABAY not defined, search-image RPC will not work.');
```
Comment (brollb) [inline src/server/rpc/procedures/pixabay/pixabay.js]: should say "Pixabay Service will not be loaded"

**Comment by brollb** [comment]
The tests are failing for this service (not just the weird issue that we have been having)

**Comment by hamidzr** [comment]
set the env variable for the service

**Comment by brollb** [comment]
yeah, I just needed to add the api key to the build env. my bad.

However, the example still looks a little unpolished... It isn't clear that the image will return to the regular size after 3 seconds...

I might actually just remove the example for now.

**Pull request merged**